lightning/src/ln/channelmanager.rs

```diff
@@ -6696,6 +6833,12 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> Writeable f
 			(11, self.probing_cookie_secret, required),
 		});
 
+		(pending_intercepted_payments.len() as u64).write(writer)?;
+		for (intercept_id, pending_intercepted_payment) in pending_intercepted_payments.iter() {
+			intercept_id.write(writer)?;
+			pending_intercepted_payment.write(writer)?;
+		}
+
```

Comment:
not sure where exactly this should be written. seemed like last was a good idea.